ElementX/Sources/Services/Audio/Player/AudioPlayer.swift

```diff
@@ -129,6 +133,38 @@ class AudioPlayer: NSObject, AudioPlayerProtocol {
     
     // MARK: - Private
     
+    private func setupAudioSession() {
+        releaseAudioSessionTask = nil
+        do {
+            try audioSession.setCategory(AVAudioSession.Category.playback)
+            try audioSession.setActive(true)
+        } catch {
+            MXLog.error("Could not redirect audio playback to speakers.")
+        }
+    }
+    
+    private func releaseAudioSession(after timeInterval: TimeInterval) {
+        guard !deinitInProgress else {
+            releaseAudioSession()
+            return
+        }
+        releaseAudioSessionTask = Task { [weak self] in
+            try? await Task.sleep(for: .seconds(timeInterval))
+            guard !Task.isCancelled else { return }
+            guard let self else { return }
+            self.releaseAudioSession()
+        }
+    }
+    
+    private func releaseAudioSession() {
+        releaseAudioSessionTask?.cancel()
+        releaseAudioSessionTask = nil
```

Comment:
As its using `@CancellableTask` we don't need to call cancel, that will happen automatically.
```suggestion
        releaseAudioSessionTask = nil
```